ui/src/model/skill.ts

```diff
@@ -0,0 +1,15 @@
+export const allSkills = [
+  "ART_2D",
+  "ART_3D",
+  "CODE",
+  "DESIGN_PRODUCTION",
+  "SOUND_MUSIC",
+  "TESTING_SUPPORT",
+  "TEAM_LEAD",
+  "OTHER",
+] as const;
+
+export type Skill = typeof allSkills[number];
```

Comment:
Out of interest, why `number` here instead of `string`?

Reply:
`allSkills` is an array, so its keys are numbers! Essentially, `typeof allStrings[number]` means "give me the type I would get if I indexed `allStrings` with a number, like `allStrings[0]`"

Reply:
Aaaah, of course! There was me thinking it was the value type, wondering if this was a hangover from the bitwise mask 😂